examples/behaviors/basic/actions/unselect-all/indicator/index.xml.njk

```diff
@@ -0,0 +1,42 @@
+{% from 'macros/button/index.xml.njk' import button %}
+{% from 'macros/description/index.xml.njk' import description %}
+
+{{ description('When selecting with delay, you can use indicators.') }}
+
+<select-multiple id="id_sm_indicator" name="sm" style="select">
```

Comment:
I think there's an opportunity to reuse this common markup (and associated styles) across all examples of select-multiple behaviors

Reply:
yes, all the examples use common styles defined in https://github.com/Instawork/hyperview/pull/904/files#diff-f510acce2d03d8e603547ee73aec6e6802a4da558929545f954f63f27711a22f.
added a macro for re-using markup